<launch>
  <arg name="cam_2" default="camera_2" />
  <arg name="bag_file_path" default="$(find gige_cam_driver)/bagfiles/cam2.bag" />

  <node pkg="gige_cam_driver" type="camera_node.py" name="$(arg cam_2)" output="screen">
      <param name="device_id" value="1"/>
      <param name="camera_manager" value="$(arg cam_2)"/>
      <param name="calibration_file" value="$(find gige_cam_driver)/config/camera_info_cam2.yaml"/>
   </node>

  <node name="rosbag_record"
  pkg="rosbag" type="record" respawn="false" output="screen"
  
  args=" --duration=2 --output-name='$(arg bag_file_path)' /camera_2/camera_info /camera_2/image_raw" />
</launch>
